packages/front/src/chess/chess.lib.ts

```diff
@@ -0,0 +1,114 @@
+import { BLACK, CHESS_BOARD_SIZE, WHITE } from './chess.constants'
+import {
+  EChessPiece,
+  TChessBoard,
+  TChessCastlingAbility,
+  TChessPieceColor,
+  TChessSquare,
+  TChessSquareIndex,
+  TChessState
+} from './chess.models'
+
+export function fenToChessPiece(fenPiece: EChessPiece): EChessPiece {
+  return fenPiece
+}
+
+export function fenToCastlingAbility(
```

Comment:
The function name won't open up to someone unfamiliar to chess. Makes sense in the context though.